src/core/mount.c

```diff
@@ -427,34 +427,21 @@ static bool mount_is_extrinsic(Mount *m) {
         return false;
 }
 
-static int mount_add_default_dependencies(Mount *m) {
+static int mount_add_default_ordering_dependencies(
+                Mount *m,
+                MountParameters *p,
+                UnitDependencyMask mask) {
```

Comment:
would have preferred if the refactoring of introduceing this new function and the other change would have been two commits, it#s so hard to review otheriwse.